lib/bokken_web/templates/email/guardian_event_reminder.html.eex

```diff
@@ -0,0 +1,103 @@
+  <table class="wrapper" style="border-collapse: collapse;table-layout: fixed;min-width: 320px;width: 100%;background-color: #fff;" role="presentation" cellspacing="0" cellpadding="0">
+    <tbody>
+      <tr>
+        <td>
+          <div>
+            <div class="layout one-col fixed-width stack" style="Margin: 0 auto;max-width: 600px;min-width: 320px; width: 320px;width: calc(28000% - 167400px);overflow-wrap: break-word;word-wrap: break-word;word-break: break-word;">
+              <div class="layout__inner" style="border-collapse: collapse;display: table;width: 100%;background-color: #ffffff;">
+                <div class="column" style="text-align: left;color: #000;font-size: 16px;line-height: 24px;font-family: Avenir,sans-serif;">
+
+                  <div style="Margin-left: 20px;Margin-right: 20px;">
+                    <div style="mso-line-height-rule: exactly;line-height: 23px;font-size: 1px;">&nbsp;</div>
+                  </div>
+
+                  <div style="Margin-left: 20px;Margin-right: 20px;">
+                    <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+                      <h1 style="Margin-top: 16px;Margin-bottom: 20px; text-align:center">
+                        <%= @event.title %>
+                      </h1>
+                    </div>
+                  </div>
+
+                  <div style="Margin-left: 20px;Margin-right: 20px;">
+                    <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+                      <ul style="Margin-top: 16px;Margin-bottom: 20px; text-align:center; list-style: none; padding: 0;">
+                        <li>📅 <%= "#{@event.start_time.day}/#{@event.start_time.month}/#{@event.start_time.year}"%> </li>
+                        <li>⏱ <%="#{@event.start_time.hour}:#{@event.start_time.minute}" %> - <%="#{@event.end_time.hour}:#{@event.end_time.minute}" %>  </li>
```

Comment:
What about adding this condition? This way, we won't have the hour being displayed as "10:0", for example.
Same for all the other templates.
```suggestion
			<%= if @event.start_time.minute < 10 do %>
                          <li>⏱ <%="#{@event.start_time.hour}:0#{@event.start_time.minute}" %> - <%="#{@event.end_time.hour}:#{@event.end_time.minute}" %>  </li>
                      <% else %>
                        <li>⏱ <%="#{@event.start_time.hour}:#{@event.start_time.minute}" %> - <%="#{@event.end_time.hour}:#{@event.end_time.minute}" %>  </li>
                      <% end %>
```